### PR TITLE
Allow unequal options for source and target

### PR DESCRIPTION
**Pull request by bierik**
It is now possible to apply different options to either the drag and drop interaction.

### CODE REVIEW

**Review by lvonlanthen — COMMENTED** [review]
👍🏼  - Two ideas:

1. In the readme, would it make sense to remove the "Options" section and rather document the available options within the "drag" and "move" sections? As they are not entirely identical: `drag` accepts `source` and `target` while `move` ignores `position`, `x` and `y` but accepts `deltaX` and `deltaY`. I think this would make the differences more explicit.
2. I think it would make sense to add a changelog to mention the new options. Also the fact that the default `position` is now [`center`](https://docs.cypress.io/api/commands/trigger#Arguments) rather than `top` might be a handy information in a changelog for users updating the package.

**Comment by bierik** [comment]
@lvonlanthen Changelog: https://github.com/4teamwork/cypress-drag-drop/pull/79

File: README.md [hunk 1]
```diff
@@ -77,70 +77,64 @@ describe('Dragtest', () => {
...
 
 # Options
 
-## position
+The options are directly passed to the `trigger` method from Cypress.
+So, all options from https://docs.cypress.io/api/commands/trigger#Arguments are possible.
 
-Decide at which position the element should be grabbed, moved and dropped.
+During the drag and drop interaction, two elements are involved. The source element being dragged and the drop target.
+In order to decide what options should be applied to the source or target, the options object can be devided in `source` and `target`. Options that are not specific to the source or target are applied to both the source and the target.
```
[the remark this answers is not in the document]
Comment (bierik) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -77,70 +77,64 @@ describe('Dragtest', () => {
...
 The command also accepts all the [common options](#options).
+But because `move` makes use of `clientX` and `clientY`, arguments such as `position`, `x` and `y` will have no effect as documented under https://docs.cypress.io/api/commands/trigger#Arguments.
 
 ```javascript
 describe('Dragtest', () => {
   it('should dragndrop', () => {
     cy.visit('/yourpage')
 
-    cy.get('.sourceitem').move({ x: 100, y: 100, ...options })
+    cy.get('.sourceitem').move({ deltaX: 100, deltaX: 100, ...options })
```
[the remark this answers is not in the document]
Comment (bierik) [reply README.md]: done

**Comment by bierik** [comment]
@lvonlanthen Could you have another look at the readme, pls?

**Review by lvonlanthen — APPROVED** [review]
🎉

**Pull request merged**